internal/core/src/index/NgramInvertedIndex.h

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2019-2020 Zilliz. All rights reserved.
```

Comment:
this header seems to be outdated

Reply:
what should it be? I just copied from some other recently added files.

Reply:
never mind, we can use -2025, but there are too many files using 2020.. we should do a batch update later